discussion.md

```diff
@@ -0,0 +1,54 @@
+#Discussion 
```

Comment:
* Need to add an space between **# Text** so it can be rendered properly. Please apply this for all titles, subtitles.
* Be consistent when listing question, i.e.: use only one question mark and do not use extra space for it.
* Are you using a markdown viewer? If don't please do so, in order to first check that the current view is the one you want to achieve.

Reply:
Pycharm editor render Markdown without spaces also. Refactored for GitHub Rendering

Reply:
Pycharm markdown render without spaces also. Refactored for GitHub